packages/react-query-devtools/src/Explorer.tsx

```diff
@@ -29,6 +30,55 @@ export const ExpandButton = styled('button', {
   padding: 0,
 })
 
+enum CopyState {
+  NoCopy,
+  SuccessCopy,
+  ErrorCopy,
+}
+
+export const CopyButton = ({ value }: { value: unknown }) => {
+  const [copyState, setCopyState] = React.useState<CopyState>(CopyState.NoCopy)
+
+  return (
+    <button
+      onClick={() => {
+        navigator.clipboard.writeText(superjson.stringify(value)).then(
+          () => {
+            setCopyState(CopyState.SuccessCopy)
+            setTimeout(() => {
+              setCopyState(CopyState.NoCopy)
+            }, 1500)
+          },
+          (err) => {
+            console.error('Failed to copy: ', err)
+            setCopyState(CopyState.ErrorCopy)
+            setTimeout(() => {
+              setCopyState(CopyState.NoCopy)
+            }, 1500)
+          },
+        )}
+      }
```

Comment:
looks good, but we're looking at a mini-state-machine here already. The user shouldn't be able to click the button while it is in the `SuccessCopy` or `ErrorCopy` state - only when it is in the `NoCopy` state. I would probably just attach the click handler when we are in that state.